x/cdp/client/cli/query.go

```diff
@@ -157,6 +158,49 @@ $ %s query %s cdps-by-ratio uatom 1.5
 	}
 }
 
+// QueryCdpDepositsCmd returns the command handler for querying the deposits of a particular cdp
+func QueryCdpDepositsCmd(queryRoute string, cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "cdp deposits [owner-addr] [collateral-name]",
```

Comment:
Looks like cobra doesn't like individual commands being more than one word, can't get this to run from the cli. I think it gets confused between this and the `cdp` command.